### DISCUSSION TITLE
Disable check DNS resolution

### DISCUSSION THREAD

**Discussion by BrunoSRibeiro** [YOURLS/YOURLS#3470] [Q&A]
Hello!

There some way to disable check DNS resolution the long url when I'm trying shorting URL, causing slow inputs.
I tryed using Static Title plugin, but the inputs is slow.
If I comment all lines in resolv.conf, the inputs is fastest.

Thanks

**Comment by ozh** [comment]
https://github.com/YOURLS/YOURLS/issues/1966#issuecomment-148905749

**Reply by BrunoSRibeiro** [reply]
Thanks for reply.

I added the plugin using your code, really dont get the title but the shorting was slowly.
I testing add the URL https://www.google.com, follow differences:

Plugin activated
![plugin](https://user-images.githubusercontent.com/21297879/206919603-57c18456-b9e0-4199-8e5e-54a19add219d.png)

Using only plugin 
![yourls2](https://user-images.githubusercontent.com/21297879/206919388-5f62f3bf-1b51-4ce3-bd6e-6e47a4ed28e7.png)

Commenting all lines in resolv.conf
![yourls1](https://user-images.githubusercontent.com/21297879/206919385-a3b2b9e5-6f86-48da-87d4-98f31bdab02f.png)

![yourls3](https://user-images.githubusercontent.com/21297879/206919941-05f19fea-eb1f-4ccf-8911-8fe13453bec5.png)

Thanks

**Reply by ozh** [reply]
Then the problem is somewhere else. Dunno where, this could be very config dependant ...
